dbfit-java/core/src/main/java/dbfit/api/DbTable.java

```diff
@@ -30,8 +30,8 @@ public DbTable(DBEnvironment dbEnvironment, String tableName)
 
     public StatementExecution buildPreparedStatement(
```

Comment:
I'm not even sure about the `StatementExecution` class name to be honest! The `Execution` bit doesn't quite sound right.